benchmarks/dream-challenge/README.md

```diff
@@ -0,0 +1,19 @@
+### DREAM Somatic Mutation Calling Challenge Benchmarks
```

Comment:
Will do, the cluster command line works pretty easily, but I can add instructions on launching a cluster from the command line or running the full benchmark and getting rid of the cluster in a separate PR.